spec/factories.rb

```diff
@@ -3,7 +3,7 @@
   factory :person do
     first_name 'John'
     last_name 'Doe'
-    email "john.doe#{Random.rand(1000)}@example.com"
+    sequence(:email) { |n| "john.doe#{n}@example.com" }
```

Comment:
Alright found the solution for this.

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#dynamic-attributes